ippserver/http_transport.py

```diff
@@ -121,9 +122,9 @@ def recv_body(self):
 
 	def send_headers(self, status='200 OK', content_type='text/plain'):
 		self._wfile.write(b'\r\n'.join((
-			b'HTTP/1.1 ' + status,
+			b'HTTP/1.1 ' + bytes(status, "utf-8"),
```

Comment:
`bytes()` only takes one argument in python2, so probably you want to do `status.encode("utf8")` instead